README.md

```diff
@@ -157,6 +157,13 @@ These are satRdays organiser groups / locations that have already run a satRday
 ## New satRdays in October - December 2020
 ### Europe [0/3 slots filled]
 ### US [0/3 slots filled]
```

Comment:
Can you pls update the slots filled?

Reply:
Looks like @malcolmbarrett beat me to it in #53 

https://github.com/satRdays/eventscalendar/commit/8d0899a640e0188f3268998f4a82c5ac571b1f2a#diff-04c6e90faac2675aa89e2176d2eec7d8R154